MD5SUMS

```diff
@@ -0,0 +1,10 @@
+6e70876751dd04accd96a95e08f3d0fe *mbdump-cdstubs.tar.bz2
```

Comment:
Some extra files got accidentally committed here (SHA256SUMS too).

Reply:
Whoops. That'd be from my testing the issues with MBS-12400. Fixed.